apps/core/scenes/search/PlacePicker/PlacePickerContent.js

```diff
@@ -0,0 +1,116 @@
+// @flow
+
+// TODO this is just the temporary Picker will be replaced by the OptionPicker form the 'universal-components'
+
+import { graphql, createRefetchContainer } from '@kiwicom/margarita-relay';
+import { type RelayRefetchProp } from 'react-relay'; // eslint-disable-line no-restricted-imports
```

Comment:
Please export this from `@kiwicom/margarita-relay` and import it from there.

Reply:
👍

Reply:
check if is OK like that, please

Reply:
Looks good yeah 👍